### ISSUE TITLE
Restarting app error (ruby 3.1.0): cannot load such file -- rexml/document

### ISSUE THREAD

**Issue by lemig** [phusion/passenger#2415]
Hi

When I do:
```
passenger-config restart-app
```
I have the error:  cannot load such file -- rexml/document
This happens with ruby 3.1.0. It's ok with ruby 2.6.0.

The raison is REXML is no longer in the Ruby Starting Library starting in ruby 3.0.0.
I suggest adding this to Gemfile:
```
if (Gem.available?("rexml"))
  gem "rexml"
end
```

**Comment by CamJN** [comment]
That doesn't work: `[!] There was an error parsing 'Gemfile': undefined method 'available?' for Gem:Module. Bundler cannot continue.`

**Comment by CamJN** [comment]
should be fixed in 6.0.14
